lib/ae_mdw/names.ex

```diff
@@ -408,10 +408,17 @@ defmodule AeMdw.Names do
   end
 
   defp expand_txi(state, bi_txi, opts) do
-    if Keyword.get(opts, :expand?, false) do
-      Txs.fetch!(state, Format.bi_txi_txi(bi_txi))
-    else
-      bi_txi
+    txi = Format.bi_txi_txi(bi_txi)
+
+    cond do
+      Keyword.get(opts, :expand?, false) ->
+        Txs.fetch!(state, txi)
+
+      Keyword.get(opts, :tx_hash?, false) ->
+        Enc.encode(:tx_hash, Txs.txi_to_hash(state, txi))
```

Comment:
It would be good for the user to get some kind of exception like [this](https://github.com/aeternity/ae_mdw/blob/98e180459cbfcbd2a69a1b9fc17367e9f4f350af/lib/ae_mdw/names.ex#L98) otherwise the param gets ignored silently. This would indicate that it should be used one param or the other.

Reply:
Done, added this validation on the `PaginationPlug` so we didn't have to on every top module